applications/asset_tracker/src/main.c

```diff
@@ -41,6 +41,7 @@ LOG_MODULE_REGISTER(asset_tracker, CONFIG_ASSET_TRACKER_LOG_LEVEL);
 
 #define CALIBRATION_PRESS_DURATION 	K_SECONDS(5)
 #define CLOUD_CONNACK_WAIT_DURATION	K_SECONDS(CONFIG_CLOUD_WAIT_DURATION)
+#define CONNECT_ERR_REBOOT_WAIT_MINUTES (5)
```

Comment:
It would be useful to have this as a Kconfig option. Some carrier have requirements on how often reconnects are allowed. Having a Kconfig allows us to make overlay files with carrier specific configs.